### ISSUE TITLE
Changes mado on SPFx project not visible in Teams

### ISSUE THREAD

**Issue by bhoomesh-spe** [MicrosoftDocs/msteams-docs#11448]
### Steps to reproduce

I have created the SPFx project and integrated that project into the Teams project

### Expected behavior

When I made the change in the SPFx project the changes were not reflected in the Teams after generating and deploying/uploading a new package in Microsoft Teams. 

### Actual behavior

The changes should be reflected to the Teams after uploading a new package in Teams.

### Error details

There are no any errors while execute below commands

gulp clean
gulp build --ship
gulp bundle --ship
gulp package-solution --ship

**Comment by sayali-MSFT** [comment]
@bhoomesh-spe -
Thanks for reporting your issue.
We will check this at our end and will get back to you.

**Comment by sagarMehta1907** [comment]
@sayali-MSFT 

Yes, we are generating an SPFx project using a Yeoman generator.

Already, we have checked all the updates you have mentioned above.

**Comment by sayali-MSFT** [comment]
@sagarMehta1907 -ok. We will try this from our end and let you know the update.

**Comment by sagarMehta1907** [comment]
When we run the application using the bellow command from the toolkit locally.

![image](https://github.com/user-attachments/assets/10224f8d-1c64-41af-9dfd-46d5101691b7)

And, Allow us to "Load debug scripts", we will see the new updates.

![image](https://github.com/user-attachments/assets/0e70f173-9820-4b18-b155-39ae6804794d)

But, while we generate a package using the command below and try to upload a zip file to the teams manually, we cannot see the updates.

![image](https://github.com/user-attachments/assets/509e6c05-97de-4604-9cd4-1301485f6995)

**Comment by sayali-MSFT** [comment]
@bhoomesh-spe - We tried the sample below, and it is working fine on our end.
Could you please follow these steps- https://learn.microsoft.com/en-us/microsoftteams/platform/sbs-gs-spfx?tabs=vscode%2Cviscode&tutorial-step=3

**Comment by sagarMehta1907** [comment]
@sayali-MSFT 

Thanks for your reply.

Yes, it works according to the link.
But it did not work after the production build.
For example, I have completed all the development in my SPFx app and now I want to publish the app into tams.
At that time if we generate a team's app zip it's content only manifest.json file.
So, How can I give my SPFx app update means package file?

**Comment by sayali-MSFT** [comment]
Please have look into this document-

**Comment by sagarMehta1907** [comment]
@sayali-MSFT 

It looks like we don't need to create a separate app for the teams. We can manage this in our SPFx app only.
However, I want to publish my app into the team's app store for the end users.
We have already published our app in the SharePoint store.

**Comment by sayali-MSFT** [comment]
Could you please confirm if your issue has resolved or still looking for any help?

**Comment by bhoomesh-spe** [comment]
Still I am facing the same issue. Please refer to the below screenshot.
![Screenshot 2024-09-09 113632](https://github.com/user-attachments/assets/940e7e6d-a8c0-469f-8b37-313347af889d)
